src/Controller/CommonAjaxControllerWhitelist.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2023 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Controller;
+
+use Glpi\Form\Form;
+
+/**
+ * Whitelist for classes allowed to be used with the CommonAjaxController class
+ */
+final class CommonAjaxControllerWhitelist
```

Comment:
I may not have a global view, but a complete class for this only method is really required?

Reply:
We should audit the GLPI classes to ensure that `CommonDBTM::can()` method is secure enough to be able to remove this list. Indeed, `CommonDBTM::can()` should be secure enough to allow any classes to be manipulated via the `CommonAjaxController`.
Please add a task in the forms feature backlog so we can do it later.